### ISSUE TITLE
why the "day" themes use black toolbar background in newui

### ISSUE THREAD

**Issue by mysmlz** [JetBrains/rider-theme-pack#18]
It's too ugly!!!
Are you sure this black toolbar background is not designed solely for the "night" themes?
![ugly](https://tva3.sinaimg.cn/large/ab4cb34aly1fepi2sz2xuj20c80c8dgd.jpg)

**Comment by sashka304** [comment]
I support this topic, please make for "day" theme a light background.

**Comment by dg00GT00** [comment]
I agree. We need something like the "Light with Light Header" theme.
